uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/CheatSheetsTest.java

```diff
@@ -0,0 +1,197 @@
+			+ "the next page, you can select specify the target environment for deploying "
+			+ "your new project and choose a specific Camel version. On the last page, you "
+			+ "can select from a list of available templates. Please select the template "
+			+ "Simple log using Spring Boot - Spring DSL from the list.Click on Finish to "
+			+ "finally create your new project.";
+
+	@BeforeClass
+	public static void setWorkspace() {
+	}
+
+	@AfterClass
+	public static void cleanWorkspace() {
+		new CleanWorkspaceRequirement().fulfill();
+	}
+
+	/**
+	 * <p>
+	 * Fuse Integration Project cheat sheet test
+	 * </p>
+	 * <b>Steps</b>
+	 * <ol>
+	 * <li>Open Help -> Cheat Sheets -> Red Hat Fuse -> Create a Fuse integration project</li>
+	 * <li>Check if texts in all sections are actual.</li>
+	 * <li>Activate available HyperLinks and wait for action.</li>
+	 * <li>Check according actions.</li>
+	 * <li>Go through New Project Wizard.</li>
+	 * </ol>
+	 */
+	@Test
+	public void testCheatSheets() {
+
+		boolean hasJava8 = hasJava8Available();
+
+		/*
+		 * Open Help -> Cheat Sheets -> Red Hat Fuse -> Create a Fuse...
+		 */
+		new ShellMenuItem(new WorkbenchShell(), "Help", "Cheat Sheets...").select();
+		Shell shell = new DefaultShell("Cheat Sheet Selection");
+		DefaultTree tree = new DefaultTree(shell);
+		TreeItem item = tree.getItem("Red Hat Fuse");
+		item.expand();
+		item.getItems();
+		List<TreeItem> items = new DefaultTree().getItems();
+		items = item.getItems();
+		TreeItem toSelect = items.get(0);
+		toSelect.select();
+		new PushButton("OK").click();
+
+		/*
+		 * Open new Workbench View for Cheat Sheets
+		 */
+		WorkbenchView cheatsheets = new WorkbenchView("Cheat Sheets");
+		cheatsheets.open();
+
+		/*
+		 * Introduction part 
+		 * After activating 'Click to Begin' introductionLink text changes to 'Click to Restart'.
+		 */
+		assertTrue("Introduction text not matching.",
+				(new DefaultFormText(new DefaultSection("Introduction")).getText()).equals(INTRODUCTION));
+		DefaultHyperlink introductionLink = new DefaultHyperlink(cheatsheets, "Click to Begin");
+		introductionLink.activate();
+		new WaitUntil(new HyperLinkHasText(introductionLink, "Click to Restart"));
+		assertTrue(
+				"Introduction hyperlink was not fulfilled correctly. Link text should be changed to Click to Restart.",
+				(introductionLink.getText().equals("Click to Restart")));
+
+		/*
+		 * Open the Fuse Integration Perspective part 
+		 * After activating 'Click to perform' perspectiveLink text changes to 'Click to redo'.
+		 */
+		assertTrue("Perspective text not matching.",
+				(new DefaultFormText(new DefaultSection("Open the Fuse Integration Perspective")).getText())
+						.equals(PERSPECTIVE));
+		DefaultHyperlink perspectiveLink = new DefaultHyperlink(cheatsheets, "Click to perform");
+		perspectiveLink.activate();
+
+		new WaitUntil(new HyperLinkHasText(perspectiveLink, "Click to redo"));
```

Comment:
please specify default timeouts for wait until methods - e.g. `new WaitUntil(new HyperLinkHasText(introductionLink, "Click to Restart"), TimePeriod.DEFAULT);`